.gitignore

```diff
@@ -3,7 +3,8 @@ kokkos/
 kokkosKernels/
 prototypes/
 doc/_build/
-PennyLane_Lightning.egg-info/
+PennyLane_Lightning_Qubit.egg-info/
+PennyLane_Lightning_Kokkos.egg-info/
```

Comment:
Should we lower-case these names in the setup.py? What is the standard python way for choosing names? I think snake_case is likely the best path.

Reply:
Done!